colors/nord.vim

```diff
@@ -797,3 +797,56 @@ function! NordPalette() abort
   let ret["nord3_bright"] = s:nord3_gui_bright
   return ret
 endfunction
+
+" Treesitter
+if has("nvim") && exists("g:nord_enable_treesitter")
+    !hi link TSError Error
+    !hi link TSPunctDelimiter  Delimiter
+    !hi link TSPunctBracket  Delimiter
+    !hi link TSPunctSpecial Delimiter
+    !hi link TSConstant Constant
+    !hi link TSConstBuiltin Constant
+    !hi link TSConstMacro Constant
+    !hi link TSStringRegex SpecialChar
+    !hi link TSString String
+    !hi link TSStringEscape String
+    !hi link TSCharacter Character
+    !hi link TSNumber Number
+    !hi link TSBoolean Boolean
+    !hi link TSFloat Float
+    !hi link TSAnnotation Structure
+    !hi link TSAttribute Attributes
```

Comment:
I'm not quite sure which syntax elements will be affected by this since the [docs are also still missing][1], but more granular adjustments can be done later on after TS is in use for this plugin and users might report theme inconsistencies. For now lets uses the [default plugin link to `PreProc`][2]. There is also currently no `Attributes` group defined which would not work anyway.

```suggestion
```

[1]: https://github.com/nvim-treesitter/nvim-treesitter/blob/3413581/doc/nvim-treesitter.txt#L455
[2]: https://github.com/nvim-treesitter/nvim-treesitter/blob/054c2a57/plugin/nvim-treesitter.vim#L55